apps/dashboard/components/virtual-table.tsx

```
@@ -51,10 +53,71 @@ export function VirtualTable<T>({
 }: VirtualTableProps<T>) {
   const parentRef = useRef<HTMLDivElement>(null);
   const tableRef = useRef<HTMLDivElement>(null);
+  const containerRef = useRef<HTMLDivElement>(null);
   const [tableDistanceToTop, setTableDistanceToTop] = useState(0);
+  const [fixedHeight, setFixedHeight] = useState(0);
+
+  // We have to lock the wrapper div at layout, otherwise causes weird scrolling issues.
+  useScrollLock({
+    autoLock: true,
+    lockTarget:
+      typeof window !== undefined
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal
Unsafe fix: Compare the result of `typeof` with a valid type name


(lint/suspicious/useValidTypeof)